src/engine/MultiColumnJoin.h

```diff
@@ -24,6 +24,8 @@ class MultiColumnJoin : public Operation {
   MultiColumnJoin(QueryExecutionContext* qec,
                   std::shared_ptr<QueryExecutionTree> t1,
                   std::shared_ptr<QueryExecutionTree> t2);
+  // Only used for testing
+  using Operation::Operation;
```

Comment:
You really shouldn't.
We try to get rid of these "only for testing, everything points to nullptr and falls apart" overloads.
It has gotten much easier to setup proper inputs for testing.